yarn.lock

```diff
@@ -1,310 +1,1330 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
we went from ~300LOC to ~1600LOC 🤯 

I see new dependencies, that I guess aren't needed? 

is it because of `"packageManager": "yarn@4.4.1",` ?

Reply:
yes, it seems to be adding some dev dependencies, not sure exactly why yet

Reply:
This happened due to a recurrent issue with fsevents and yarn, which ends up installing node-gyp even if it's not listed as dependency. sass package depends on chokidar, which depended on fsevents. Coincidentally, few days ago they removed this dependency from chokidar, so upgrading sass solves this. 😄